sdl3.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "sdl3",
+    "modules": [
+        {
+            "name": "SDL3",
+            "builddir": true,
+            "buildsystem": "cmake-ninja",
+            "cleanup": [
+                "/include",
+                "/lib/*.la",
+                "/lib/*.a",
+                "/lib/cmake",
+                "/share/aclocal",
+                "/lib/pkgconfig"
+            ],
+            "config-opts": [
+                "-DCMAKE_VERBOSE_MAKEFILE=ON",
+                "-DCMAKE_BUILD_TYPE:STRING=Release",
+                "-DCMAKE_INSTALL_LIBDIR:PATH=lib"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/libsdl-org/SDL/releases/download/release-3.2.8/SDL3-3.2.8.tar.gz",
+                    "sha256": "13388fabb361de768ecdf2b65e52bb27d1054cae6ccb6942ba926e378e00db03"
+                }
+            ]
+        },
+        {
+            "name": "sdl3_image",
+            "builddir": true,
+            "buildsystem": "cmake-ninja",
+            "cleanup": [
+                "/include",
+                "/lib/*.la",
+                "/lib/*.a",
+                "/lib/cmake",
+                "/share/aclocal",
+                "/lib/pkgconfig"
+            ],
+            "config-opts": [
+                "-DCMAKE_VERBOSE_MAKEFILE=ON",
+                "-DCMAKE_BUILD_TYPE:STRING=Release",
+                "-DCMAKE_INSTALL_LIBDIR:PATH=lib"
+            ],
+            "sources":[
+                {
+                    "type": "archive",
+                    "url": "https://github.com/libsdl-org/SDL_image/releases/download/release-3.2.4/SDL3_image-3.2.4.tar.gz",
```

Comment:
I have not noticed this earlier but why are we mix and matching SDL3 versions between the core library vs the extensions?

3.2.4 here vs 3.2.8 for core